admin-dev/themes/new-theme/js/pages/shipping-preferences/index.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
```

Comment:
```suggestion
const {$} = window;

```

Isn't missing?

Reply:
ping @NeOMakinG (otherwise everything is ok for me)

Reply:
No, @NeOMakinG asked us to convert the .js to .ts when we convert it there is an error. After remove no problem.

I have already seen with @NeOMakinG in private but  wait he confirm.

I add a very nice commit message for this ( private joke ) 

![Capture d’écran de 2021-10-04 18-49-19](https://user-images.githubusercontent.com/5064590/135891789-f6eb4822-e9ec-402d-946e-c12b7558dce0.png)

Reply:
Yep, as it's not an es module, declaring jQuery is useless because it's available globally